src/components/Signup/components/container/index.jsx

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import '../../../login/styles/index.css';
+
+export class Signup extends Component {
+  state = {
+    firstname: '',
+    lastname: '',
+    email: '',
+    username: '',
+    password: '',
+    confirmpassword: '',
+  };
+  render() {
+    const {
+      firstname, lastname, email, username, password, confirmpassword,
```

Comment:
Use camel casing, confirmPassword for your variables.

Reply:
+1 on this

Reply:
+1 on this

Reply:
@dondrzzy , @kmunene , @alexmochu I made the requested changes but I had not respond to the comments